Fresh volume module implementation feature

Dolgolyov4000 opened the pull request:
Most recent version of volume module.

vviolka committed:
GUI

Dolgolyov4000 committed:
Merge pull request #1 from Nebyshinets4000/master

GUI

Commit:
Global code refactoring (stage 1)

Commit:
Surface scene 3D fix.

Commit:
Dramatical UI changes.

Base classes refactoring.
Volume render technique integration attempt.

Commit:
Attempt of cuboid remapping integration - still need some fixes.

Commit:
Commented error code section.

senkevich4000 committed:
New default camera settings.

Dolgolyov4000 committed:
Merge pull request #2 from senkevich4000/master

New default camera settings.

Commit:
Added volume camera adaptivity.

Commit:
Shaders little fix.

Added workspace loading popup.

Commit:
Fragment shader optimization.

Slicing implemented.

senkevich4000 committed:
Basic implementation of intensities remapping and normals calculation.

senkevich4000 committed:
Fixed error with intensity opactiy & global opactiy. Refactoring: u_ prefix for all uniforms, remove redundant log messages.

senkevich4000 committed:
Spots recalculation with respect to size factor parameter.

Dolgolyov4000 committed:
Merge pull request #3 from senkevich4000/master

Cuboids mapping & normals calculation

Commit:
Slicing controls update.

Volume vertex shader improvement.

Commit:
Merge branch 'master' of https://github.com/Dolgolyov4000/ili

senkevich4000 committed:
Shader loading with ShaderLoader. Better logging in Remapping Processor.

senkevich4000 committed:
Update intensities volume on molecule change.

senkevich4000 committed:
Merge branch 'master' of https://github.com/dolgolyov4000/ili

senkevich4000 committed:
Update fragment and vertex shaders. Remove volumeshaers.js

Commit:
Shaders update.

senkevich4000 committed:
Merge with Dolgolyov4000/ili master

senkevich4000 committed:
Fix typo.

Commit:
Export fix.

Commit:
Added texture caches.

Commit:
Added workers buffers transfer semanticks in order to decrease memory traffic.
Added lazy normals compute.

Dolgolyov4000 committed:
Merge branch 'master' into master

Dolgolyov4000 committed:
Merge pull request #4 from senkevich4000/master

Shader loading & molecule selection

Commit:
Fixed merge errors.

Commit:
Added remapping on size scale changed and slider timeout.
Fixed volume data cache classes.

senkevich4000 committed:
Merge with dolgolyov master.

Commit:
Fixed 2D rendering - dummy camera options.

Commit:
Added cuboid border opacity support.

senkevich4000 committed:
Merge with Dolgolyov master.

Commit:
Added ellipsoid mapping mode.
Shape reset fixed.
Added Bansai demo.
Added mapping enabled flag.

Commit:
Surface module raycaster fixed.

Commit:
View2D dummy camera fix.

sonicXconst1 committed:
Merge branch 'master' of https://github.com/dolgolyov4000/ili

senkevich4000 committed:
SpotsController: new intensity change handler. No remapping on min/max change.

senkevich4000 committed:
Remove redundant log messages!

Dolgolyov4000 committed:
Merge pull request #5 from senkevich4000/master

Remove invalid remapping on min/max change.

Commit:
Fixed address line parsing.

Commit:
Merge branch 'master' of https://github.com/Dolgolyov4000/ili

Commit:
Volume documentation fixed.
Documentation templates separated.
View3D upgraded.

senkevich4000 committed:
Fix typos in requirejs paths.

senkevich4000 committed:
Merge with Dolgolyov4000 master branch.

Dolgolyov4000 committed:
Merge pull request #6 from senkevich4000/master

Fix typos in requriejs configuration.

senkevich4000 committed:
Fix bug with molecules switching. Improve performance on mapping tab.

Dolgolyov4000 committed:
Merge pull request #7 from senkevich4000/master

Fix bug with molecules switching. Improve performance on mapping tab.

senkevich4000 committed:
Implement intensity threshold. Add optional example: Example1.

senkevich4000 committed:
Merge branch 'master' of https://github.com/dolgolyov4000/ili

senkevich4000 committed:
Fix hotkeys. The problem was with leak of MapSelectorBase dependency in AppBase. Also Firefox does not capture onkeypress event, replaced with keydown.

senkevich4000 committed:
Remove ellipsoid mode.

Dolgolyov4000 committed:
Merge pull request #8 from senkevich4000/master

Intensity Threshold

Commit:
Coordinates adjustment added.

Commit:
Merge branch 'master' of https://github.com/Dolgolyov4000/ili

Commit:
Coordinates adjustment fix.

richardgoater commented:
@Dolgolyov4000 I think this has all been merged, unless you know of anything new since last night?

Dolgolyov4000 commented:
Well, you're right. Seems like I missed something.

richardgoater commented:
I think it's my fault, I squashed the previous PR. I could undo that if you
like? Or you could rebase your master.

On Thu, 11 Feb 2021, 15:44 Dolgolyov4000, <notifications@github.com> wrote:

> Well, you're right. Seems like I missed something.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/MolecularCartography/ili/pull/94#issuecomment-777589362>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAK5BWNDACJLJK6MCDF5V4TS6P3ONANCNFSM4XOOO5WQ>
> .
>

Dolgolyov4000 committed:
Merge branch 'MolecularCartography:master' into master

vviolka committed:
orientation cube

vviolka committed:
added interactive orientation cube

vviolka committed:
Layout of legend and orientation widget

Dolgolyov4000 committed:
Merge pull request #9 from Nebyshinets4000/master

Orientation cube + layout of legend

vviolka committed:
Added transfer function control

vviolka committed:
Transfer function control: density based ticks

vviolka committed:
Transfer function control: density based ticks

vviolka committed:
Move Drawing and CoordsTransformer to separate files

vviolka committed:
Fix ticks drawing

vviolka committed:
Added ObjectCache

vviolka committed:
Added ObjectCache

Dolgolyov4000 committed:
Merge pull request #10 from Nebyshinets4000/master

Transfer function control

vviolka committed:
Interactivity system

vviolka committed:
Update documentation

vviolka committed:
Fixed pan

Dolgolyov4000 committed:
Merge pull request #11 from Nebyshinets4000/master

Interactivity system

Commit:
Refactor/new features

= Refactored existing code base
+ Added volume section
+ Added volume lego view
+ Added settings

Commit:
Merge branch 'master' of https://github.com/Dolgolyov4000/ili

vviolka committed:
Moved some files to common

vviolka committed:
Pointer capturing on transfer function control

vviolka committed:
Fixed group box arrows

vviolka committed:
Fixed default view animation

Dolgolyov4000 committed:
Merge pull request #12 from Nebyshinets4000/master

Some fixes

Commit:
Dramatical changes

= Upgrades of interactivity system
+ Camera animation fixed
+ Surface default view and animation added
+ Vizualiation improved
- Orbit controls deleted

Commit:
Merge branch 'master' of https://github.com/Dolgolyov4000/ili

Commit:
Major improvements.

Major improvements.
Shader upgrade.
Common infrastructure addition.
Surface module functionality fixed after refactoring.
